CHANGELOG/CHANGELOG-2.6.0.md

```diff
@@ -0,0 +1,154 @@
+# Release notes for v2.6.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v2.5.0
+
+## Changes by Kind
+
+### Bug or Regression
+
+- Corrects the handler used to serve http requests, the /healthz and /debug/ endpoints are available now. (#190, @cnmcavoy)
```

Comment:
Can you format it like this?
https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/master/CHANGELOG/CHANGELOG-6.1.md

Reply:
done, please take a look again